File: library/src/main/java/me/proxer/library/enums/LengthBound.java
package me.proxer.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum for selecting the length bound.
 *
 * @author Ruben Gees
 */
public enum LengthBound {
    @Json(name = "up")HIGHER,
    @Json(name = "down")LOWER
}

File: library/src/main/kotlin/me/proxer/library/enums/LengthBound.kt
package me.proxer.library.enums

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Enum for selecting the length bound.
 *
 * @author Ruben Gees
 */
@JsonClass(generateAdapter = false)
enum class LengthBound {

    @Json(name = "up")
    HIGHER,

    @Json(name = "down")
    LOWER
}
